src/App/Validator/ExportJob/Update.php

```diff
@@ -20,13 +20,101 @@ class Update extends Validator
 
     protected function getRules()
     {
-        return [
+
+        return array_merge([
             'id' => [
                 ['numeric'],
             ],
             'entity_type' => [
                 ['in_array', [':value', ['post']]],
-            ]
-        ];
+            ],
+        ], $this->getHxlRules());
+    }
+
+    /**
+     * @return array
+     * Return hxl rules array if Feature hxl is enabled, empty array otherwise
+     */
+    private function getHxlRules()
+    {
+        $hxl_rules = [];
+        if (Features::isEnabled('hxl')) {
+            $hxl_rules = [
+                'send_to_hdx' => [
+                    [[$this, 'sendToBrowserIsFalse'], [':value', ':fulldata', ':validation']],
+                ],
+                'send_to_browser' => [
+                    [[$this, 'sendToHDXIsFalse'], [':value', ':fulldata', ':validation']],
+                ],
+                'include_hxl' => [
+                    [[$this, 'trueIfSendToHDXIsTrue'], [':value', ':fulldata', ':validation']],
+                ]
+            ];
+        }
+        return $hxl_rules;
+    }
+
+    /**
+     * @param $validation
+     * @param $value
+     * @param $fullData
+     * @return bool
+     */
+    public function trueIfSendToHDXIsTrue($value, $fullData, $validation)
+    {
+        if ($fullData['send_to_hdx'] === true && $value === false) {
+            $validation->error('include_hxl', 'includeHXLShouldBeTrue');
+        }
+        return true;
+    }
+
+    /**
+     * @param $validation
+     * @param $value
+     * @param $fullData
+     * @return bool
+     */
+    public function sendToBrowserIsFalse($value, $fullData, $validation)
+    {
+        if (!$this->isOppositeBool($fullData['send_to_hdx'], $fullData['send_to_browser'])) {
+            $validation->error('send_to_hdx', 'sendToHDXShouldBeTrue');
+        }
+        return true;
+    }
+
+    /**
+     * @param $validation
+     * @param $value
+     * @param $fullData
+     * @return bool
+     */
+    public function sendToHDXIsFalse($value, $fullData, $validation)
+    {
+        if (!$this->isOppositeBool($fullData['send_to_browser'], $fullData['send_to_hdx'])) {
+            $validation->error('send_to_browser', 'sendToBrowserShouldBeTrue');
+        }
+        return true;
+    }
+
+    /**
+     * @param $first
+     * @param $second
+     * @return bool
+     */
+    private function isOppositeBool($first, $second)
```

Comment:
Reading the above, I just assumed `isOppositeBool` was already defined in Validator. 
Future: we might as well share this helper there or in a trait or somewhere.

Reply:
yep absolutely. Adding to my list of things to do after release, I'll create a ticket